### DISCUSSION TITLE
Feature Request: Give PoIygon Admin!!!!!

### DISCUSSION THREAD

**Discussion by PoIygon** [CluckCluckChicken/Magnifier#104] [big chungus]
**Feature**
The feature should be to give @PoIygon on magnifier admin!!!!! He told me he will be a good admin and we won’t do anything bad!!,!!
**Additional context**
He will be a good admin

**Comment by davwat** [comment]
you would say that wouldnt you

**Comment by FunctionalMetatable** [comment]
i tried asking already but I didn't get it :(((((

**Comment by davwat** [comment]
ok fine ill make you admin
![magnifier poiygon admin](https://user-images.githubusercontent.com/46281840/120446167-34dc9180-c381-11eb-87a8-9aa43c4fb822.png)
_oh no my fingers slipped and i clicked on the wrong boolean_
[/joke]

**Comment by PoIygon** [comment]
Ban me pls

**Comment by davwat** [comment]
or not?

**Comment by PoIygon** [comment]
> or not?

or not?

**Comment by davwat** [comment]
so you want magnifier reactions to suddenly stop working for you because the only time the ban message displays is when you try to log in (#86)?

**Comment by JackyBorderCollie** [comment]
can you make me admin too?
